src/webhooks/index.ts

```diff
@@ -9,38 +11,47 @@ import { kafkactlWebhook } from './kafka-control-plane/kafka-control-plane';
 import { sentryOptionsWebhook } from './sentry-options/sentry-options';
 import { webpackWebhook } from './webpack/webpack';
 
+type WebhookHandler = (
+  request: FastifyRequest<any>,
+  reply: FastifyReply
+) => Promise<void>;
+
 // Error handling wrapper function
 export async function handleRoute(
-  handler,
+  handler: WebhookHandler,
   request: FastifyRequest,
-  reply: FastifyReply
+  reply: FastifyReply,
+  name: string
```

Comment:
Something about this makes me wish there was a better way, but this makes sense and works